python_transport/wirepas_gateway/transport_service.py

```diff
@@ -173,6 +235,9 @@ class TransportService(BusClient):
     # Period in s to check for black hole issue
     MONITORING_BUFFERING_PERIOD_S = 1
 
+    # Period in s to send gateway status to the backend
+    SEND_GATEWAY_STATUS_PERIOD_S = 20
```

Comment:
see comment above for a default value.

Reply:
perhaps we would also want to configure this from environment variable (i..e add a parameter to argument parser tool.